files/default/dism_features.rb

```diff
@@ -0,0 +1,45 @@
+#
+# Author:: Wade Peacock <wade.peacock@visioncritical.com>
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Ohai.plugin(:DismFeatures) do
```

Comment:
I'd love if we could get this added to Ohai itself and then we can skip the installation of the plugin based on that released version of Ohai. Shipping this via the cookbook is useful to existing clients, but going forward we should pull it into core so we can avoid that hassle of performing the installation.